design/005-Wire-Encoding-Abstraction.md

```diff
@@ -0,0 +1,118 @@
+# Proposal: Abstraction of Wire Messages Encoding
+
+* Author(s): Sebastian Stammler (@sebastianst)
```

Comment:
* Status: accept
 
 Can you add this line. It helps us keep track of whether a proposal was accepted or rejected.